docs/source/azure/advanced/azurefunctions.rst

```diff
@@ -36,12 +36,13 @@ Execution in Azure functions comes with a default set of configurations for the
 resources. To override these settings you must set 'provision-options' with one of the following
 keys:
 
+- groupName (default: cloud-custodian)
 - location (default: West US 2)
 - appInsightsLocation (default: West US 2)
-- servicePlanName (default: cloud-custodian)
-- sku (default: Basic)
-- skuCode (default: B1)
-- workerSize (default: 0)
+- servicePlanName (default: cloud-custodian-plan)
+- skuTier (default: Basic)
```

Comment:
If we just give the sku code (B1, S2, etc) is that enough? Do we even need the tier?